### PR TITLE
STIFI-61 TTS echo route

### PR DESCRIPTION
**Pull request by SmariF89**
This modifies the echo route so that it will _audibly_ echo caller's speech using TTS.
Before it only echoed the caller by logging what he said in console.

With the conclusion of this implementation we have set up a nice base for a continued implementation of an IVR-client for SDiFI with a continuous flow of user speech and TTS responses in their respective directions.

### CODE REVIEW

**Comment by SmariF89** [comment]
@rkjaran Klára e.t.v. að fara yfir og tvinna https://github.com/SDiFI/headless/pull/2 og endurgrunna svo þessa við master.

File: src/app.py [hunk 1]
```diff
@@ -38,48 +43,96 @@ def handle_error(err):
...
+                elif data["event"] == "closed":
+                    current_app.logger.info("CLOSE: {}".format(message))
+                    ws.close()
+                    break
+                elif data["event"] == "mark":
+                    current_app.logger.info("MARK")
+                else:
+                    current_app.logger.info(f"UNHANDLED_MSG_TYPE: {data['event']}")
+        except:
+            current_app.logger.exception("Something went wrong while receiving data.")
```
Comment (rkjaran) [inline src/app.py:78]: re`raise`a

File: src/app.py [hunk 2]
```diff
@@ -38,48 +43,96 @@ def handle_error(err):
...
+                    )
+                )
+            else:
+                current_app.logger.warning(
+                    f"TTS_REQ_FAILURE_CODE: {tts_resp.status_code}"
+                )
+        except:
+            current_app.logger.exception(
+                "Something went wrong while processing ASR data.",
+            )
```
Comment (rkjaran) [inline src/app.py:137]: re`raise`a

**Review by rkjaran — CHANGES_REQUESTED** [review]
Smá nitpick.

File: src/app.py (hunk 1 above)
Comment (SmariF89) [reply src/app.py:78]: Græjað.

**Pull request merged**